kokoro/config/test/ops_agent/rockylinux9.gcl

```diff
@@ -1,7 +1,8 @@
 import 'common.gcl' as common
+import '../image_lists.gcl' as image_lists
 
 config build = common.ops_agent_test {
   params {
-    platforms = ['rocky-linux-9']
+    platforms = image_lists.centos.distros.rockylinux9.release
```

Comment:
This should end with `.presubmit` not `.release`. The GCL file you are editing is for the ops agent presubmit tests, not the release tests. (release tests cover a broader range of distros).